packages/djangoflow_app/lib/src/presentation/builders/app_cubit_builder.dart

```diff
@@ -7,6 +7,7 @@ class AppCubitBuilder extends BlocBuilder<AppCubit, AppState> {
     required super.builder,
     BlocBuilderCondition<AppState>? buildWhen,
   }) : super(
+          bloc: AppCubit.instance,
```

Comment:
Why did you add this hardcoded this `bloc` here? Can you explain?

Reply:
We should not hardcode any blocs unless we absolutely need it. Why would you want to add an extra line of code in multiple places when you can provide it by the provider once and the blocBuilders or listeners can find the correct bloc automatically?